==== triton/lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVMPass.cpp#17 - /google/src/cloud/ecg/mlir_2aec7083ada09c8b8a0aad79492cbedcf8f9fbb7_1704830674/triton/lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVMPass.cpp ====
# action=edit type=text
--- triton/lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVMPass.cpp	2024-01-05 23:46:53.000000000 -0800
+++ triton/lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVMPass.cpp	2024-01-09 12:13:15.000000000 -0800
@@ -226,7 +226,8 @@
     }
     // Set an attribute for maxntidx, it could be used in latter LLVM codegen
     // for `nvvm.annotation` metadata.
-    newFuncOp->setAttr("nvvm.maxntid", rewriter.getI32ArrayAttr(32 * numWarps));
+    newFuncOp->setAttr("nvvm.maxntid",
+                       rewriter.getDenseI32ArrayAttr(32 * numWarps));
     // The call graph is updated by mapping the old function to the new one.
     allocation.mapFuncOp(funcOp, newFuncOp);
 
